packages/ag-charts-community/src/chart/axis/axis.ts

```diff
@@ -301,10 +301,12 @@ export abstract class Axis<S extends Scale<D, number, TickInterval<S>> = Scale<a
 
     private attachCrossLine(crossLine: CrossLine) {
         this.crossLineGroup.appendChild(crossLine.group);
+        if (crossLine.labelGroup) this.crossLineGroup.appendChild(crossLine.labelGroup);
```

Comment:
Is there any risk that `labelGroup` will be added/removed between the attach and detach? (It might be better to just have a fixed/static group on the `CrossLine` type, and always add/remove it - which is the approach we've taken elsewhere to give better consistency guarantees).

Reply:
I wouldn't have thought so, but I'll change it to match the rest